react-interactive/src/ConfigurableTable/ConfigurableTable.tsx

```diff
@@ -117,233 +124,291 @@ interface TableProps<T> {
+            sortKey: string;
+            disableAdd: boolean;
+        }
+        
+        function ColumnSelection(props: IColSelectionProps) {
+            return (
+                <>
+                <div className="row">
+                <div className="col-4">
+                {props.columns.map((c: IColDesc, i: number) =>
+                    i % 3 == 0 ? (
+                        <CheckBox
+                        Label={c.Label}
+                        Field={'Enabled'}
+                        Record={c}
+                        Setter={() => props.onChange(c.Key)}
+                        key={c.Key}
+                        Disabled={c.Key == props.sortKey || (props.disableAdd && !c.Enabled)}
+                        Help={
+                            c.Key == props.sortKey
+                            ? 'The Table is currently sorted by this column so it cannot be hidden.'
+                            : undefined
+                        }
+                        />
+                    ) : null,
+                )}
+                </div>
+                <div className="col-4">
+                {props.columns.map((c: IColDesc, i: number) =>
+                    i % 3 == 1 ? (
+                        <CheckBox
+                        Label={c.Label}
+                        Field={'Enabled'}
+                        Record={c}
+                        Setter={() => props.onChange(c.Key)}
+                        key={c.Key}
+                        Disabled={c.Key == props.sortKey || (props.disableAdd && !c.Enabled)}
+                        Help={
+                            c.Key == props.sortKey
+                            ? 'The Table is currently sorted by this column so it cannot be hidden.'
+                            : undefined
+                        }
+                        />
+                    ) : null,
+                )}
+                </div>
+                <div className="col-4">
+                {props.columns.map((c: IColDesc, i: number) =>
+                    i % 3 == 2 ? (
+                        <CheckBox
+                        Label={c.Label}
+                        Field={'Enabled'}
+                        Record={c}
+                        Setter={() => props.onChange(c.Key)}
+                        key={c.Key}
+                        Disabled={c.Key == props.sortKey || (props.disableAdd && !c.Enabled)}
+                        Help={
+                            c.Key == props.sortKey
+                            ? 'The Table is currently sorted by this column so it cannot be hidden.'
+                            : undefined
+                        }
+                        />
+                    ) : null,
+                )}
+                </div>
+                </div>
+                <div>
+                {props.disableAdd ? (
+                    <Alert AlertColor='alert-primary' className="mb-0 mt-2">Additional columns disabled due to table size.</Alert>
```

Comment:
There is no className prop on the alert component in gemstone. We can add one or you could use the Style prop -- `Style={{marginBottom: 0, marginTop: '0.5em'}}`